2020-01-22 00:02:09,587 ERROR: Exception on / [GET] [in C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\routes.py", line 14, in about
    return render_template('about.html', title='About')
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\../templates\about.html", line 1, in <module>
    {% extends base.html %}
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'base' is undefined
2020-01-22 00:02:09,926 ERROR: Exception on / [GET] [in C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\routes.py", line 14, in about
    return render_template('about.html', title='About')
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\../templates\about.html", line 1, in <module>
    {% extends base.html %}
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'base' is undefined
2020-01-22 00:06:16,698 INFO: Microblog startup [in C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\__init__.py:47]
2020-01-22 00:06:20,209 ERROR: Exception on / [GET] [in C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py:1890]
Traceback (most recent call last):
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\routes.py", line 15, in about
    return render_template('about.html', title='About')
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\app\../templates\about.html", line 1, in <module>
    {% extends base.html %}
  File "C:\Users\Esther Goldberg\PycharmProjects\Esther Goldberg Final Project\venv\lib\site-packages\jinja2\environment.py", line 430, in getattr
    return getattr(obj, attribute)
jinja2.exceptions.UndefinedError: 'base' is undefined
